framework/db/Command.php

```diff
@@ -537,6 +541,7 @@ public function createTable($table, $columns, $options = null)
     public function renameTable($table, $newName)
     {
         $sql = $this->db->getQueryBuilder()->renameTable($table, $newName);
+        $this->requireTableSchemaRefreshment($table);
```

Comment:
Make sense. 

I think `_refreshTableName` should be set null when the command is being reused. Otherwise whenever you are reusing the command for other purposes, it will mark the table schema as invalid. Not very common, but should be similar to the use case you gave.